lib/screens/v2/widget_instance/alert.ex

```diff
@@ -107,6 +107,7 @@ defmodule Screens.V2.WidgetInstance.Alert do
         _ -> route_id
       end
     end)
+    |> Enum.take(2)
```

Comment:
Just checking: is this something you agreed on with Laina and design?

Reply:
No, and after checking Adam's designs again I just noticed that the route pills are small enough to fit 3—I'll update styles so that we aren't using _exactly_ the same route pills as what's shown in departure rows.
Will also double check with Laina and design on what should happen if there are more than 3 pills to show on a single flex zone alert.

Reply:
After checking with Adam on this, here's what we'll do:
- up to 2 affected routes → full-size route pills
- 3 affected routes → small route pills
- more than 3 affected routes → collapse to single full-size mode pill
- affected entity is whole stop, or whole mode → single full-size mode pill

Reply:
(Originally he had cases of >3 pills wrap onto a second line)